staging/src/k8s.io/component-base/featuregate/feature_gate.go

```diff
@@ -542,6 +544,16 @@ func (f *featureGate) GetAllVersioned() map[Feature]VersionedSpecs {
 	return retval
 }
 
+// GetVersionedSpecs gets the VersionedSpecs information for a feature.
+func (f *featureGate) GetVersionedSpecs(key Feature) (VersionedSpecs, error) {
```

Comment:
I would avoid adding this to the interface just for one caller, when it can do `specs, ok := f.GetAllVersioned()[key]` itself